### DISCUSSION TITLE
Is there a way to get a random sample?

### DISCUSSION THREAD

**Discussion by argenisleon** [h2oai/datatable#3197] [Q&A] [answered]
Looking at the docs, it seems there is no function to get a random sample of a dataframe.
Any way workaround to accomplish this?

**Comment by oleksiyskononenko** [comment, marked as answer]
Well, that's just a matter of generating a list of random indices from `0` to `nrows - 1` and then using this list in the `[i, j, ...]` selector as `DT[list_of_random_indices, :]`. The `list_of_random_indices` can be easily generated by using the standard Python module `random`. For instance, `random.sample()` could be used for this purpose: https://docs.python.org/3/library/random.html#random.sample

If the random samples are needed for K-fold cross validation, then please take a look at the parallel `kfold_random()` function, that is a part of the `datatable.models`: https://datatable.readthedocs.io/en/latest/api/models/kfold_random.html
